config/checkstyle/checkstyle.xml

```diff
@@ -162,7 +162,7 @@
         </module>
         <module name="AbbreviationAsWordInName">
             <property name="ignoreFinal" value="false"/>
-            <property name="allowedAbbreviationLength" value="1"/>
+            <property name="allowedAbbreviationLength" value="3"/>
```

Comment:
I was thinking of whether we can disable the rule specifically for the two MDC classes, not globally for the project, do you know if it's possible?

Reply:
@yurishkuro , Yes I added these classes to the suppressions.xml file
the build still fails in codecov I don't know why